universal-application-tool-0.0.1/build.sbt

```diff
@@ -104,17 +112,40 @@ lazy val root = (project in file("."))
     Compile / packageDoc / publishArtifact := false,
     Compile / doc / sources := Seq.empty
   )
+  .settings(excludeTailwindGeneration: _*)
+
+lazy val excludeTailwindGeneration = Seq(watchSources := {
```

Comment:
suggestion (blocking): please add a comment explaining why this is excluded

Reply:
done